x-pack/plugins/task_manager/server/saved_objects/model_versions/task_model_versions.ts

```diff
@@ -35,4 +35,18 @@ export const taskModelVersions: SavedObjectsModelVersionMap = {
       create: taskSchemaV2,
     },
   },
+  '3': {
+    changes: [
+      {
+        type: 'mappings_addition',
+        addedMappings: {
+          priority: { type: 'integer' },
```

Comment:
\o/ this move will probably help us with https://github.com/elastic/kibana/issues/194627. Would we / could we start persisting priority for all tasks on update and eventually more to a non-scripted sort in the task claimer?